packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -68,12 +68,21 @@ export function getBeaconPoolApi({
     },
 
     async getPoolAttesterSlashings() {
+      const fork = chain.config.getForkName(chain.clock.currentSlot);
+
+      if (isForkPostElectra(fork)) {
```

Comment:
we might not be able to serialize data if this is called post-electra since the limit of `attesting_indices` is higher, better to throw an error and refer to v2 api